build.gradle

```diff
@@ -12,7 +12,7 @@
  */
 
 apply plugin: 'java'
-apply plugin: 'fatjar'
+apply plugin: "eu.appsatori.fatjar"
```

Comment:
It looks like this plugin is no longer under development, per: https://github.com/musketyr/gradle-fatjar-plugin

Do you mind converting to https://github.com/johnrengelman/shadow?